vello_encoding/src/encoding.rs

```diff
@@ -415,6 +415,10 @@ impl Encoding {
             .extend_from_slice(bytemuck::bytes_of(&DrawImage {
                 xy: 0,
                 width_height: (image.width << 16) | (image.height & 0xFFFF),
+                sample_alpha: ((image.quality as u32) << 12)
+                    | ((image.x_extend as u32) << 10)
+                    | ((image.y_extend as u32) << 8)
```

Comment:
Maybe? It would be worth `debug_assert`ing that these all fit. Since these enums are exhaustive, having an exhaustive match `#[cfg(test)]`ed somewhere would also be sufficient.